flags/urls.py

```diff
@@ -78,7 +78,6 @@ def url_patterns(self):
                 pattern.default_args,
                 pattern.name,
             )
-            print(pattern, flagged_pattern)
```

Comment:
Woops! This was printing in tests and for any users. Added in #101.

Reply:
Good catch!